Update draft-retana-idr-bgp-quic-02.xml

gregbo opened the pull request:
spelling corrections, grammar corrections, and clarifications

gregbo committed:
Update draft-retana-idr-bgp-quic-02.xml

spelling corrections, grammar corrections, and clarifications

gregbo committed:
Update draft-retana-idr-bgp-quic-02.xml

minor correction

jhaas-pfrc commented on draft-retana-idr-bgp-quic-02.xml:
```diff
@@ -130,7 +130,7 @@
        control of the connection.
      </t>
      <t>
-        This document specifies the procedures for BGP to use QUIC as a transport protocol with a mechanism to carry Network Layer protocols (AFI/SAFI) over individual streams.  The Network layer protocols are identified using a combination of Address Family (AF) and Subsequent Address Family (SAFI), as described in <xref target="RFC4760"/>. These per-AFI/SAFI streams (function channels) and the  ...
```
We'll be wanting to keep AFI here from 4760.
The bare "address family" is regularly used in bgp-land to imply the afi/safi pairing, but isn't terribly well supported by the core documents.

gregbo replied:
I'm not sure what you mean, because I replaced "AF" with "AFI" in my commit.  If you want me to undo the replacement, OK, but it seems awkward to me to use "AF" and "SAFI" in that sentence.

jhaas-pfrc replied:
Insufficient coffee first thing in the morning error.  This is correct.  Thanks.

jhaas-pfrc commented on draft-retana-idr-bgp-quic-02.xml:
```diff
@@ -254,19 +254,19 @@
-            <t>When one BoQ speaker is configured as client, and the other side is configured as server, no collision will happen. If the other side initiates a QUIC connection, a QUIC CONNECTION_CLOSE frame with error code APPLICATION_ERROR MUST be sent.</t>          
-            <t>When a BoQ speaker is configured as any or server, it MUST accept the QUIC connection initiated by the other BoQ speaker.</t>
+            <t>When one BoQ speaker is configured as a client, and the other side is configured as a server, no collision will happen. If the other side initiates a QUIC connection, a QUIC CONNECTION_CLOSE frame with error code APPLICATION_ERROR MUST be sent.</t>          
+            <t>When a BoQ speaker is configured as "any" or a server, it MUST accept the QUIC connection initiated by the other BoQ speaker.</t>
```
as "any" or as a server

jhaas-pfrc commented on draft-retana-idr-bgp-quic-02.xml:
```diff
@@ -364,7 +364,7 @@
             <tr>
               <td>68</td>
               <td>Multisession BGP Capability</td>
-              <td>draft-ietf-idr-bgp- multisession</td>
```
These splits are, sadly, deliberate.  Without them, the ascii form of the table doesn't render in a pleasing way.

If you have thoughts about how to influence column rendering for the generated table to split on a hyphen and suggest column width in the xml input, we're happy for suggestions.

gregbo replied:
Hmmm.  I wasn't displeased by the ascii (txt) form of the table produced (by the author-tools site).  What do others think?  Given how important BGP (and QUIC) are to Internet technology, I'd hate to see the draft held up over something like this.

jhaas-pfrc replied:
Would you cache your rendered version with this change somewhere we can see?  I'm basing my opinion of this on my locally executed xml2rfc invocation.

gregbo replied:
It's at https://github.com/gregbo/BGP-QUIC/blob/patch-1/draft-retana-idr-bgp-quic-02.txt

jhaas-pfrc replied:
Thanks for demonstrating that it's my local issue rather than an issue with the xml.  

Let's implement your changes here.

That said, please remove the .txt from the change set.  The authors would prefer to manage that artifact as part of our release process.

gregbo replied:
done

jhaas-pfrc reviewed (commented):
Thanks for your review comments.  Happy to merge after the discussed points are covered.

gregbo committed:
Update draft-retana-idr-bgp-quic-02.xml

minor correction

gregbo committed:
Add files via upload

generated from my commits to draft-retina-idr-bgp-quic-02.xml by author-tools.ietf.org

gregbo committed:
Delete draft-retana-idr-bgp-quic-02.txt

no longer needed

jhaas-pfrc reviewed (approved):
Looks good. Thanks!
